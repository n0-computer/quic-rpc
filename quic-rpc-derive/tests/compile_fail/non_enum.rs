use quic_rpc_derive::rpc_requests;

#[rpc_requests(Service)]
struct Foo;

fn main() {}