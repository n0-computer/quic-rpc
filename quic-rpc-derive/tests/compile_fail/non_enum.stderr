error: RpcRequests can only be applied to enums
 --> tests/compile_fail/non_enum.rs:4:1
  |
4 | struct Foo;
  | ^^^^^^
